doc/deploy.md

```diff
@@ -81,57 +179,23 @@ If you like Certbot, please consider supporting our work by:
 - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
 ```
 
-## Configuring Docker
-
-To ensure that Docker containers has access to networking, you may need to set up IPv4 port forwarding on your server machine:
-
-```sh
-sudo sysctl net.ipv4.ip_forward=1
-sudo sysctl -p
-sudo service docker restart
-```
-
-## Run Envoy
-
-### TODO make docker image with both envoy and quickfeed and expose ports needed etc
-
-```sh
-% brew install tmux overmind 
-% Procfile
-```
-
-### Generate Envoy Configuration File
-
-```sh
-% cd $QUICKFEED
-% cp doc/scripts/envs.sh quickfeed-env.sh
-# Edit quickfeed-env.sh according to your domain
-% source quickfeed-env.sh
-% make envoy-config
-```
-
-Note that you may need to disable NGINX on Ubuntu:
+### Run Envoy
 
-```sh
-# Temporarily stop running NGINX
-% sudo systemctl stop nginx
-# Permanently disable NGINX
-% sudo systemctl disable nginx
-```
+Please choose one of the options below to run envoy in your system:
 
-### Running with Locally Installed Envoy (macOS homebrew)
+1. Running with Locally Installed Envoy (macOS homebrew)
 
 ```sh
-% sudo envoy -c $ENVOY_CONFIG &
+% sudo envoy -c $QUICKFEED/envoy/envoy.yaml &
```

Comment:
I removed it because when building the docker image, the `envoy.yaml` config must be inside the context of the Dockerfile, otherwise commands like ADD or COPY do not work.
In the case of the envoy, the context is the folder `$QUICKFEED/envoy`. So, if you specify a config in a folder outside that one, the docker builder will not be able to find it. I could not find an easy way to switch between the two approaches other than requiring users to create the `envoy.yaml` inside `$QUICKFEED/envoy`.

One possible alternative would be to have a script that copies the config to the right place before building the image.